docs/integration/integration-more.md

```diff
@@ -165,3 +149,5 @@ Alephium implemented the same 51% attack protection as [Ravencoin](https://tronb
 ### What is the maximum reorg/rollback possible on Alephium?
 
 The maximum allowed reorg size is 100 blocks. It is recommend to check 110 blocks for higher safety.
+
```

Comment:
Perhaps we can mention that after the Rhone upgrade, the maximum reorg size is 200.